rfc/Resolver/CategoryForHeadless.ts

```diff
@@ -1,62 +1,134 @@
-import { Breadcrumb } from "./Breadcrumb";
-import { ProductForCategoryPageHeadless } from "./ProductForCategoryPageHeadless";
 import { CmsPage } from "../../packages/shopware-6-client/src/interfaces/models/content/cms/CmsPage";
-import { Media } from "../../packages/shopware-6-client/src/interfaces/models/content/media/Media";
-
-// PLACEHOLDERS IN CMS PAGES OF SHOPWARE 6
-//
-// category.type
-// category.name
-// category.externalLink
-// category.description
-// category.media.alt
-// category.media.title
-// category.media.url
-// category.media
 
 export interface CategoryForHeadless {
-  availableSorting: [Sort];
-  availableFiltering: [Filter];
-
-  id: string;
-  name: string | null;
-  description: string;
-  childCount: number;
-
-  parentId: string | null;
-  breadcrumb: Breadcrumb[];
-  level: number;
-
-  children: CategoryForHeadless[] | null;
-  products: ProductForCategoryPageHeadless[] | null;
+  availableSorting: Sort // sample below within avaiableSortingResponseExample
+  availableFiltering: Array<OptionFilter | RangeFilter | BooleanFilter> // sample below within avaiableFiltersResponseExample
+  cmsPage: CmsPage | null
+}
 
-  cmsPage: CmsPage | null;
+// <SORTING>
+interface Sort {
+  code: string
+  value: string // field name to sort by
```

Comment:
Isn't it same as `code`? Can they be different?

Reply:
value could also contain "-" minus sign. can be prefixed with it. but now I don't see any use case when we could use it instead of the value...